packages/epics/src/profile/hooks/use-me.ts

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import React from 'react';
+import useSWR from 'swr';
+import { useAuthentication } from '@hypha-platform/authentication';
+import { Person } from '@hypha-platform/storage-postgres';
+
+export const useMe = (): { person: Person; isLoading: boolean } => {
```

Comment:
It's not correct. This hook can return `Person | undefined`, right?

Reply:
> It's not correct. This hook can return `Person | undefined`, right?

Fixed